src/OpenLoco/GameCommands/GameCommands.h

```diff
@@ -842,17 +842,17 @@ namespace OpenLoco::GameCommands
             , rotation(regs.dl)
             , type(regs.bh)
             , unk(regs.dh)
-            , primaryColour(regs.bp & 0xFF)
-            , secondaryColour((regs.bp >> 8) & 0xFF)
+            , primaryColour(static_cast<Colour>(regs.bp & 0x1F))
```

Comment:
different values?

Reply:
Yeah i felt it was better to cap it to the correct range.

Reply:
No worries, wasn't sure if it was a typo or not